library/src/main/java/io/github/douglasjunior/androidSimpleTooltip/SimpleTooltip.java

```diff
@@ -216,11 +217,14 @@ private void verifyDismissed() {
     }
 
     private void createOverlay() {
+        if (ignoreOverlay) {
+            return;
+        }
         mOverlay = mTransparentOverlay ? new View(mContext) : new OverlayView(mContext, mAnchorView, mHighlightShape, mOverlayOffset);
         if (mOverlayMatchParent)
-            mOverlay.setLayoutParams(new ViewGroup.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.MATCH_PARENT));
+            mOverlay.setLayoutParams(new ViewGroup.LayoutParams(ViewGroup.LayoutParams.WRAP_CONTENT, ViewGroup.LayoutParams.WRAP_CONTENT));
```

Comment:
I think that this change is unexpected. ¹

Reply:
Oh sorry, you're right, test data out of control 👍